app/models/dataset_record/darwin_core/taxon.rb

```diff
@@ -254,6 +254,7 @@ def import(dwc_data_attributes = {})
               excluded: 'TaxonNameClassification::Iczn::Unavailable::Excluded',
               'nomen nudum': 'TaxonNameClassification::Iczn::Unavailable::NomenNudum',
               ichnotaxon: 'TaxonNameClassification::Iczn::Fossil::Ichnotaxon',
+              fossil: 'TaxonNameClassification::Iczn::Fossil',
```

Comment:
@LordFlashmeow Fossil is basically an abstract class at this point, see `ichnotaxon` above for fossil use.